### PR TITLE
✨ Add Drone-CI as vendored schema

### PR DESCRIPTION
**Pull request by s-weigand**
This PR adds [drone-ci](https://docs.drone.io/) to the vendored schema.

The positive test examples I added are from the [pipeline docs](https://docs.drone.io/pipeline/overview/).

And the negative one just has a bad value for the `type` of the pipeline.

As per the docs there also is an [AWS pipeline](https://docs.drone.io/pipeline/aws/overview/) but the current schema does not support it (tested with YAML LSP in vscode) so I didn't add it as a test case.

### CODE REVIEW

**Comment by s-weigand** [comment]
Also thanks for the project this is exactly what I was looking for ❤

File: .gitignore
Comment (sirosen) [inline .gitignore:9]: A quick note: I won't require this to be removed but I would prefer not to add it. I may remove it in the future (but I'm not such a curmudgeon that I'll do it right after merging "just because" :wink: ).

My opinion is that `.gitignore` files per repo should cover files produced by that repo, and that a global gitignore should be used for developer-specific things (`.idea`, `.vscode`, `.DS_Store`, etc).

**Review by sirosen — APPROVED** [review]
Awesome, thanks for contributing this and for your kind words!

I'm happy to include this and release; and everything I've double-checked lines up (e.g. the file path regex matches the schemastore catalog and the Drone CI docs).
I hope adding this was an easy and generally postiive experience!

I've left a little noncritical note about `.gitignore`, but I won't worry about it since there are no other issues to address.

**Comment by sirosen** [comment]
Oh :facepalm: I didn't let CI run before approving. Let's wait for that to confirm, and then I'll merge assuming it passes.

**Pull request merged**
